.go-tools

```diff
@@ -1,3 +1,4 @@
 github.com/oligot/go-mod-upgrade@v0.9.1
 gotest.tools/gotestsum@v1.8.2
 github.com/segmentio/golines@v0.11.0
+github.com/google/go-containerregistry/cmd/crane@v0.12.1
```

Comment:
Just curious why the switch?

Reply:
You can't go install skopeo. Crane has everything we need for this use case. Might even think about seeing if I can switch it out in mindthegap.